_assets/scss/modules/_projects.scss

```diff
@@ -25,6 +25,18 @@
     margin-bottom: 0;
     // @include col-third(false);
   }
+
+  h4 {
+    @include h4--med;
+    line-height: $unit * .875;
+    color: $black;
```

Comment:
Yes. I would remove them and change the default color and then edit which colors were affected by the change.